boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm7.dts

```diff
@@ -16,9 +16,13 @@
 	chosen {
 		zephyr,sram = &sdram0;
 		zephyr,dtcm = &dtcm;
+		zephyr,itcm = &itcm;
```

Comment:
What is this for?

Reply:
The FlexSPI driver uses the `zephyr,chosen` node for ITCM to select where to relocate the FlexSPI driver code at runtime: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/CMakeLists.txt#L32 (`CONFIG_FLASH_MCUX_FLEXSPI_XIP_MEM`) defaults to ITCM